charts/datadog/templates/_container-host-release-volumemounts.yaml

```diff
@@ -4,7 +4,7 @@
   mountPath: /host{{ .Values.datadog.systemProbe.osReleasePath | default .Values.datadog.osReleasePath }}
   mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
   readOnly: true
-{{- else }}
+{{- else if not .Values.providers.gke.autopilot}}
```

Comment:
I was thinking that `read-only` mount was "OK" on auto-pilot. 
Does the rule has changed?

Reply:
the only volume that needed read/write was the volume that store the release.json used by the logs-agent.